free-1.md

```diff
@@ -10,34 +11,57 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This DSL would have two main user bases: people who play DOTA 2, and people who are trying to develop and test new heroes (characters in the game) and items for the game. Ideally, the second group would be a subset of the first group, since to use the tool would require a good amount of knowledge of the game. The idea is to provide an text interface with very low overhead to allow people to test different hero/item combinations, givng stats like damage potential, tankiness, disable duration, etc for these combinations, as well as allowing people to test new heroes/items in the same way. 
+
+I think a language is the best way to accomplish this, as there is already a GUI for it (namely, the game itself in practice mode), but its very slow. To test things, you need to start up the game, and try a bunch of different things, all of which take time. If there were a text interface, you could save an item combination, then just loop over the heroes you want to test and produce output for all of them instantly. Further, if there were a way to save a test set (for example, damage at X level, pushing power, mana usage, farming speed) one could do mass testing using these, without having to worry about forgetting anything. Additionally, having a text interface would let you run this while in game, if there's a quick thing you need to check (you could even pause the game, check something, then unpause).
+
+DOTA 2 is a very, very hard game. There are 120+ heroes to learn, all with 4+ unique abilities, and over 100 items. As a result, when making balance changes, there are millions of interactions between abilities and items that contribute to its balance, so having a test suite of things to check (like, X hero should do more damage but have less health than Y hero) would help the developers of the game a lot to make sure that a new change doesn't mess things up. To be fair, I'm not sure what their current process looks like, its entirely possible that they already have something like this, but this would help the community be more involved, and have more informed debates about changes. 
+
+Additionally, to just the average user, this could be a great learning tool. If you want to learn how strong a certain hero is against another hero, you could use this to run damage calculations. You could also use this to test different builds (like, if I get X item before Y item, I'll farm faster but be easier to kill). As of now, the way to do this is literally just to play the game. You test different item builds, and try them in the game, most of the time leading to your death as you learn what things work and don't work. This could simoplify that process. 
 
```

Comment:
I don't know much about Dota2 (or MOBAs in general), but this seems like a great idea.  I think all competitive games that involve upgrades/stats need something like this for theorycrafting to be able to work at a large scale.